src/Innertube.ts

```diff
@@ -325,6 +325,18 @@ export default class Innertube {
     return response.data?.unseenCount || response.data?.actions?.[0].updateNotificationsUnseenCountAction?.unseenCount || 0;
   }
 
+  /**
+   * Retrieves playlists.
+   */
+  async getPlaylists() {
+    const response = await this.actions.execute(
+      BrowseEndpoint.PATH, { ...BrowseEndpoint.build({ browse_id: 'FEplaylist_aggregation' }), parse: true }
+    );
+
+    const feed = new Feed(this.actions, response);
+    return feed.playlists;
```

Comment:
@LuanRT  I just realized that perhaps we should return the feed, in case there is continuation (I don't know if there is a continuation/pagination with playlists or not)
I see this is already released in 9.4 (thanks!) so maybe it can be in 10.x if it make sense

Reply:
Looks like there is, so yup I agree.